docs/frequently_asked_questions_faq.md

```diff
@@ -1,3 +1,133 @@
-# Frequently Asked Questions (FAQ)
 
-ya know, like the kinds of questions folks like to ask. 
\ No newline at end of file
+
+# FreeMoCap FAQ's - Ongoing Draft
```

Comment:
its kinda silly, but lets define "FAQ" here 

most everyone will know what an FAQ is but, the rules are that you have to define and acronym the first time you use it